ui/src/components/Sidebar.tsx

```diff
@@ -876,7 +878,10 @@ function ExportJupyterNB() {
   const store = useContext(RepoContext);
   if (!store) throw new Error("Missing BearContext.Provider in the tree");
   const repoName = useStore(store, (state) => state.repoName);
-  const pods = useStore(store, (state) => state.pods);
+  const nodesMap = useStore(store, (state) => state.getNodesMap());
+  const nodes = Array.from<ReactflowNode>(nodesMap.values());
+  const podResults = useStore(store, (state) => state.podResults);
+  const codeMap = useStore(store, (state) => state.getCodeMap());
```

Comment:
Is the `podResults` not synced with YJS by design?

Reply:
It should be saved to Yjs. It's on my TODO list.

Reply:
> It should be saved to Yjs. It's on my TODO list.

Is there any concern to append `podResults` to `codeMap`?

Reply:
That's an option; I'm thinking about it.